sources/assets/exegol/skel/bloodhound_customqueries/merge/README.txt

```diff
@@ -0,0 +1,2 @@
+Place here your valid BloodHound customqueries files.
```

Comment:
A more specific message maybe ? "in this directory" + the extension of the file to add ?

Reply:
I renamed the file README.md, added the markdown and tried to rework the content. Hopefully it's clearer that way.